advanced/Scripts/list.sh

```diff
@@ -84,6 +84,11 @@ PoplistFile() {
     touch ${whitelist}
   fi
 
+  # Check blacklist file exists, and if not, create it
+  if [[ ! -f ${blacklist} ]]; then
```

Comment:
We should probably have `${...}` here and below

Reply:
Actually, yeah, good shout. Probably on the whitelist part above, too.

@LudovicRousseau, could you please change `${blacklist}` to `"${blacklist}"` (lines 88 + 89) and `${whitelist}` to `"${whitelist}"` (lines 83 + 84)